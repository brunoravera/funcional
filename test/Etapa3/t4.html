<html>
	<head>
		<link rel="stylesheet" href="gift.css">
	</head>
	<body>
<div class="giftComentario">
ejercicio que testea mo2short
</div>
<span class="giftNombre">
ej1</span>
<div class="giftFragmento">
Quien es verdadero</div>


<div class="giftShort">
\item <span class="giftMATH">
3=3</span>

<div class="giftFragmento">
</div>


\item <span class="giftCode">
foo == foo</span>

<div class="giftFragmento">
</div>


\item <div class="giftFragmento">
Top</div>


</div>

<div class="giftFragmento">
</div>


<div class="giftComentario">

</div>

<div class="giftComentario">
ejercicio que testea nodupMO
</div>
<span class="giftNombre">
ej2</span>
<div class="giftFragmento">
pregunta</div>


<div class="giftMO">
\item <div class="giftFragmento">
si</div>


\item <div class="giftFragmento">
noup</div>


\item <div class="giftFragmento">
no</div>


\item <div class="giftFragmento">
nop</div>


</div>

<div class="giftFragmento">
mas texto</div>


<div class="giftComentario">

</div>

<div class="giftComentario">
ejercicio que testea shortMO
</div>
<span class="giftNombre">
ej3</span>
<div class="giftFragmento">
pregunta</div>


<div class="giftMO">
\item <div class="giftFragmento">
si1</div>


\item <div class="giftFragmento">
si2</div>


\item <div class="giftFragmento">
no1</div>


\item <div class="giftFragmento">
no2</div>


</div>

<div class="giftFragmento">
mas texto</div>


<div class="giftComentario">

</div>

<div class="giftComentario">
ejercicio que testea trim
</div>
<span class="giftNombre">
ej4</span>
<div class="giftFragmento">
quien es estrictamente positivo</div>


<div class="giftMO">
\item <span class="giftMATH">
1  +0</span>

<div class="giftFragmento">
</div>


\item <div class="giftFragmento">
uno</div>


\item <div class="giftFragmento">
unidad</div>


\item <div class="giftFragmento">
diez</div>


\item <div class="giftFragmento">
</div>

<span class="giftMATH">
0</span>


</div>

<div class="giftFragmento">
mas texto</div>


<div class="giftComentario">

</div>

	</body>
</html>
