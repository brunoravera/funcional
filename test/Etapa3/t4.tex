\documentclass{gift}
\begin{document}
\begin{giftComentario}
ejercicio que testea mo2short
\end  {giftComentario}
\giftNombre{ej1}
\begin{giftFragmento}
Quien es verdadero\end  {giftFragmento}


\begin{giftShort}
\item $3=3$
\begin{giftFragmento}
\end  {giftFragmento}


\item \verb!foo == foo!
\begin{giftFragmento}
\end  {giftFragmento}


\item \begin{giftFragmento}
Top\end  {giftFragmento}


\end  {giftShort}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\begin{giftComentario}
ejercicio que testea nodupMO
\end  {giftComentario}
\giftNombre{ej2}
\begin{giftFragmento}
pregunta\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
si\end  {giftFragmento}


\item \begin{giftFragmento}
noup\end  {giftFragmento}


\item \begin{giftFragmento}
no\end  {giftFragmento}


\item \begin{giftFragmento}
nop\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
mas texto\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\begin{giftComentario}
ejercicio que testea shortMO
\end  {giftComentario}
\giftNombre{ej3}
\begin{giftFragmento}
pregunta\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
si1\end  {giftFragmento}


\item \begin{giftFragmento}
si2\end  {giftFragmento}


\item \begin{giftFragmento}
no1\end  {giftFragmento}


\item \begin{giftFragmento}
no2\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
mas texto\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\begin{giftComentario}
ejercicio que testea trim
\end  {giftComentario}
\giftNombre{ej4}
\begin{giftFragmento}
quien es estrictamente positivo\end  {giftFragmento}


\begin{giftMO}
\item $1  +0$
\begin{giftFragmento}
\end  {giftFragmento}


\item \begin{giftFragmento}
uno\end  {giftFragmento}


\item \begin{giftFragmento}
unidad\end  {giftFragmento}


\item \begin{giftFragmento}
diez\end  {giftFragmento}


\item \begin{giftFragmento}
\end  {giftFragmento}

$0$

\end  {giftMO}

\begin{giftFragmento}
mas texto\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\end{document}
