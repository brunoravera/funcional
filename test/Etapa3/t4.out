CUESTIONARIO
============

PREGUNTA ej1
COMENTARIO INICIAL: ejercicio que testea mo2short
[Q [TXT "Quien es verdadero"],A (SHORT [OK [MATH "3=3",TXT ""],OK [CODE "foo == foo",TXT ""],OK [TXT "Top"]]),Q [TXT ""]]
COMENTARIO FINAL: 
------------
PREGUNTA ej2
COMENTARIO INICIAL: ejercicio que testea nodupMO
[Q [TXT "pregunta"],A (MO [OK [TXT "si"],NOK [TXT "noup"],NOK [TXT "no"],NOK [TXT "nop"]]),Q [TXT "mas texto"]]
COMENTARIO FINAL: 
------------
PREGUNTA ej3
COMENTARIO INICIAL: ejercicio que testea shortMO
[Q [TXT "pregunta"],A (MO [OK [TXT "si1"],OK [TXT "si2"],NOK [TXT "no1"],NOK [TXT "no2"]]),Q [TXT "mas texto"]]
COMENTARIO FINAL: 
------------
PREGUNTA ej4
COMENTARIO INICIAL: ejercicio que testea trim
[Q [TXT "quien es estrictamente positivo"],A (MO [OK [MATH "1  +0",TXT ""],OK [TXT "uno"],OK [TXT "unidad"],NOK [TXT "diez"],NOK [TXT "",MATH "0"]]),Q [TXT "mas texto"]]
COMENTARIO FINAL: 
------------
